src/controllers/QueryRunner.ts

```diff
@@ -58,6 +60,8 @@ export default class QueryRunner {
         this._uri = _ownerUri;
         this._title = _editorTitle;
         this._isExecuting = false;
+        this.batchSets = [];
```

Comment:
I agree with you. I think this was leftover from before, but I'll change it to make it match.